code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -291,8 +291,7 @@
 	if(holder && holder.my_atom)
 		var/turf/open/T = get_turf(holder.my_atom)
 		if(istype(T))
-			var/datum/gas/gastype = /datum/gas/nitrogen
-			T.atmos_spawn_air("[initial(gastype.id)]=50;TEMP=2.7")
+			T.atmos_spawn_air("n2=50;TEMP=2.7")
```

Comment:
This can't use the N2 define?

Reply:
It can't be like
```suggestion
			T.atmos_spawn_air("[GAS_N2]=50;TEMP=2.7")
```

Reply:
that implies I have to do it for EVERY other case where atmos parse string is used. This belongs to another refactor